Embedded Metadata.js

```diff
@@ -563,8 +564,9 @@ function addLowQualityMetadata(doc, newItem) {
 	if(newItem.title) {
 		newItem.title = newItem.title.replace(/\s+/g, ' '); //make sure all spaces are \u0020
 		if(newItem.publicationTitle) {
-			//remove publication title from the end of title
-			var removePubTitleRegex = new RegExp('\\s*\\W\\s*'
+			//remove publication title from the end of title (see #604)
+			//this can occur if we have to doc.title, og:title etc.
+			var removePubTitleRegex = new RegExp('\\s*-\\s*'
```

Comment:
This should include other punctuation, like `|` and `:` and all the dashes (en and em).

I didn't want to try and figure out what exactly that punctuation may be, but I made the mistake of not excluding spaces from `\W`. You can do that by replacing `\\W` with `(?!\\s)\\W` We can go with explicit list of punctuation though.